docs/Configuration-Settings.md

```diff
@@ -87,6 +87,11 @@ The common configuration contains constants shared by different components.
   <td>Flag used only during tests to allow special behavior.</td>
 </tr>
 <tr>
+  <td>tachyon.thrift.stop.timeout.seconds</td>
```

Comment:
This parameter was added only for speeding up unit tests and wasn't intended to be user-facing (https://github.com/amplab/tachyon/pull/1974/files#diff-e1f168d496029608a5f4a145489ff9f6R104). I've created a separate PR to use a constant instead of putting in tachyon-defaults (https://github.com/amplab/tachyon/pull/2308).